### ISSUE TITLE
Bug: Some regions don't work with PIA

### ISSUE THREAD

**Issue by qdm12** [qdm12/private-internet-access-docker#159]
**TLDR**: *AU Sydney doesn't work sometimes with a TLS error*

1. Is this urgent?

    - [ ] Yes
    - [x] No

2. What VPN service provider are you using?

    - [x] PIA
    - [ ] Mullvad
    - [ ] Windscribe

3. What's the version of the program?

    **See the line at the top of your logs**

    `Running version latest built on ? (commit ?)`

4. What are you using to run the container?

    - [ ] Docker run
    - [x] Docker Compose
    - [ ] Kubernetes
    - [ ] Docker stack
    - [ ] Docker swarm
    - [ ] Podman
    - [ ] Other:

5. Extra information

Logs:

```log

```

Configuration file:

```yml

```

Host OS:

**Comment by qdm12** [comment]
Fixed by https://github.com/qdm12/private-internet-access-docker/commit/0710199409c680d40d746cb1c147cf31effd52d9
